airflow/utils/classes.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import absolute_import
```

Comment:
Nit: Do you think it would make sense to put the classes into their own files? better for modularization, and helps remove the vagueness of the name "classes.py"